fe/fe-core/src/main/java/com/starrocks/load/streamload/StreamLoadMgr.java

```diff
@@ -212,27 +203,27 @@ public StreamLoadTask createLoadTaskWithoutLock(Database db, String tableName, S
         return streamLoadTask;
     }
 
-    public StreamLoadTask createLoadTask(Database db, String tableName, String label, String user, String clientIp,
+    public StreamLoadTask createLoadTask(Database db, Table table, String label, String user, String clientIp,
```

Comment:
looks like these `createLoadTask` can be static function. The implementation doesn't access any member variables.

Reply:
This function is no called out of `StreamLoadMgr`, so there is no need to be `public static` currently, so I just make it private.